src/pool/srv_util.c

```diff
@@ -404,10 +404,12 @@ update_one_tgt(struct pool_map *map, struct pool_target *target,
 /*
  * Update "tgts" in "map". A new map version is generated only if actual
  * changes have been made.
+ *
  */
 int
-ds_pool_map_tgts_update(struct pool_map *map, struct pool_target_id_list *tgts,
-			int opc, bool evict_rank)
+ds_pool_map_tgts_update(struct pool_map *map, uint32_t *tgt_map_ver,
+			struct pool_target_id_list *tgts, int opc,
+			bool evict_rank)
```

Comment:
Minor: Should have added tgt_map_ver as the last parameter as it's an output-only parameter. A comment for tgt_map_ver would also be useful.

Reply:
oh, sure.